src/components/Display/Display.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Display from './Display';
+
+describe('Calculator', () => {
+    let wrapper;
+    beforeEach(() => wrapper = shallow(<Display displayValue={'0'}/>))
+
+    it('should render a div', () => {
+        expect(wrapper.find('div').length).toEqual(1);
```

Comment:
```suggestion
        expect(wrapper.find('div').length).toBe(1);
```

Personally, I would only use `toEqual()` when expecting the value of either an object or an array. While both work in fine in this case, I favor `toBe()` since it is more strict.

See the [Common Matches Section](https://jestjs.io/docs/en/using-matchers#common-matchers) of the Jest docs for more info about the difference between the two.